CMakeLists.txt

```diff
@@ -569,6 +572,10 @@ ocv_include_directories(${OPENCV_CONFIG_FILE_INCLUDE_DIR})
 #  Path for additional modules
 # ----------------------------------------------------------------------------
 set(OPENCV_EXTRA_MODULES_PATH "" CACHE PATH "Where to look for additional OpenCV modules (can be ;-separated list of paths)")
+# ----------------------------------------------------------------------------
+#  Path for Tengine modules
+# ----------------------------------------------------------------------------
+set(OPENCV_TENGINE_INSTALL_PATH "" CACHE PATH "Where to look for additional OpenCV modules (can be ;-separated list of paths)")
```

Comment:
> additional OpenCV modules

What?

Move configuration stuff into separate CMake file: cmake/OpenCVFindTengine.cmake

Such vars should have `_ROOT_DIR` suffix.
`_INSTALL_PATH` is reserved for install rules.

Reply:
done